freqtrade/strategy/interface.py

```diff
@@ -357,13 +357,13 @@ def min_roi_reached(self, trade: Trade, current_profit: float, current_time: dat
         # Check if time matches and current rate is above threshold
         trade_dur = (current_time.timestamp() - trade.open_date.timestamp()) / 60
 
-        # Get highest entry in ROI dict where key >= trade-duration
-        roi_entry = max(list(filter(lambda x: trade_dur >= x, self.minimal_roi.keys())))
+        # Get highest entry in ROI dict where key <= trade-duration
+        roi_list = list(filter(lambda x: x <= trade_dur, self.minimal_roi.keys()))
+        if not roi_list:
+            return False
+        roi_entry = max(roi_list)
```

Comment:
This whole section should be extracted into it's own function and called from both here and `backtesting`.

Reply:
Maybe a better option would be to return the roi_entry just calculated in the should_sell() function a few lines above... (in a new member of SellCheckTuple, like SellCheckTuple.data; for SellType.ROI it can be assigned roi_entry from min_roi_reached(), for SellType.STOP_LOSS and SellType.TRAILING_STOP_LOSS -- the trade.stop_loss as used in stop_loss_reached() or some other useful value...

What do you think?

Also, the comment here
```
# get next entry in min_roi > to trade duration
# Interface.py skips on trade_duration <= duration
```
says the value of min_roi here should be different from one in strategy/interface.py.
And it also contradicts with the code implemented. Maybe someone has fixed an error here but did not change the old comment. Could you review these both places where roi_entry is calculated?

Reply:
> # Interface.py skips on trade_duration <= duration

IIRC, the 2nd comment was there since previously this was done differently in interface.py - but we discovered that this can lead to odd behaviour if a unsorted ROI dict is passed in, so the 2 methods were aligned.
Probably the comment was forgotten

Reply:
Done (duplicated code extracted into a separate function)